QuantumToolbox.jl/time_evolution/fluorescence.qmd

```diff
@@ -0,0 +1,230 @@
+$$
+\Omega \cos(\omega_L t) \Big[e^{i\omega_L t}\sigma^{+} + e^{-i\omega_L t}\sigma^{-}\Big] = \frac{\Omega}{2}\Big[(1 + e^{2i\omega_L t})\sigma^{+} + (1 + e^{-2i\omega_L t})\sigma^{-}\Big]
+$$
+The time-dependent parts with frequency $2\omega_L$ are dropped, yielding
+$$
+H^\text{rot} \simeq \frac{\omega_a}{2} \sigma_z + \frac{\Omega}{2} \sigma_x - \frac{\omega_L}{2} \sigma_z = \frac{\Delta}{2} \sigma_z + \frac{\Omega}{2} \sigma_x
+$$
+where $\Delta \equiv \omega_a - \omega_L$ is the detuning between the two-level atom and the driving light. In the realistic near-resonance case $\Delta \simeq 0$, characteristic of the radiative behaviour remains. Thus, for the following demonstration, we only consider the exact resonance $\Delta = 0$. 
+
+### Dissipation dynamics
+
+We start by the interaction Hamiltonian between the thermal field and atom
+$$
+\hat{H}_{\text{a}}^\text{int} = \sum_l \alpha_l \left( \hat{b}_l + \hat{b}_l^\dagger \right) \left( \sigma^{-} + \sigma^{+} \right)
+$$
+where for the $l$-th mode
+
+- $\alpha_l$ is the coupling strength with the atom
+- $\hat{b}_l$ is the annihilation operator
+
+By applying [rotating wave approximation (RWA)](https://en.wikipedia.org/wiki/Rotating-wave_approximation) and following the standard procedure of the [Born-Markovian approximation](https://en.wikiversity.org/wiki/Open_Quantum_Systems/The_Quantum_Optical_Master_Equation), we obtain the atom dissipation rate $\gamma_0$. Consequently, the dynamics is described by the [Lindblad master equation](https://en.wikipedia.org/wiki/Lindbladian).
+$$
+\mathcal{L} = \gamma_0 n(\omega_a, T) \mathcal{D}[\sigma^{+}] + \gamma_0 [1 + n(\omega_a, T)] \mathcal{D}[\sigma^{-}]
+$$
+
+where
+
+- $n(\omega, T)$: Bose-Einstein distribution of the thermal field at temperature $T$
+- $\mathcal{D}[\cdot]$: The [Lindblad dissipator](https://en.wikipedia.org/wiki/Lindbladian) (has exactly the same expresseion in the lab frame and the rotating frame)
+
+## Code Demonstration 
+
+```{julia}
+import QuantumToolbox: QuantumObjectEvolution, n_thermal, liouvillian, sigmap, sigmam, sigmax, 
+    sigmay, sigmaz, basis, mesolve, correlation_2op_1t, spectrum_correlation_fft
+import CairoMakie: Figure, Axis, @L_str, lines!, axislegend, display, ylims!, xlims!
+```
+
+```{julia}
+Ω = 1
+γ0 = 0.05 * Ω
+KT = 0 # thermal field at zero temperature  
+
+```
+
+We define a function that returns the Liouvillian `SuperOperator` of the system.  
+```{julia}
+function liouvillian_spec(_Ω, _γ0, _KT)
+    H = _Ω/2 * sigmax()
+    c_ops = [
+        √(_γ0 * n_thermal(_Ω, _KT)) * sigmap(),
+        √(_γ0 * (1 + n_thermal(_Ω, _KT))) * sigmam(),
+    ]
+    return liouvillian(H, c_ops)    
+end
+```
+
+We first use `QuantumObject.mesolve` to solve the master equation with several observables given in `e_ops`.  
```

Comment:
Also here, `[mesolve](https://qutip.org/QuantumToolbox.jl/stable/resources/api#QuantumToolbox.mesolve)` would be better.